android-components/.buildconfig.yml

```diff
@@ -2027,6 +2027,12 @@ projects:
     - support-utils
     - tooling-lint
     - ui-icons
+  support-remotesettings:
+    description: A component to help fetch data from remote servers.
+    path: components/support/remotesettings
+    publish: true
+    upstream_dependencies:
+      - support-base
```

Comment:
I'm not really familiar with when dependencies need to be upstreamed. I would assume that it's only required if we need to expose an API from the dependency. Is that correct? What API do we need exposed from `support-base`?

Reply:
Using it for Logger now